x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_inventory/service_inventory.spec.ts

```diff
@@ -77,11 +75,14 @@ describe('When navigating to the service inventory', () => {
     cy.contains('h1', 'opbeans-node');
   });
 
-  describe('Calls APIs', () => {
+  describe.skip('Calls APIs', () => {
```

Comment:
can you leave a comment here to explain why it's skipped (and maybe reference an issue)?

Reply:
Sure, it was [failing](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/398#86dc0d3a-3063-4956-89c6-4411679867d8/513-1503) while waiting for the API calls to finish